lib/v2/producthunt/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/today': ['Fatpandac'],
```

Comment:
```suggestion
    '/today': ['miaoyafeng', 'Fatpandac'],
```
Please add back old route author's name according to docs